### ISSUE TITLE
Icons fail to display

### ISSUE THREAD

**Issue by petarvucetin** [Armaell/home-assistant-custom-icons-loader#1]
Nice work. 

Followed the instructions via HACS. Added custom_icons: to my config. I can see the folder under custom_components. Added svg image to root (custom_icons) and to folder my-icons under custom icons.

I can see in the log that it has been loaded:
```
2020-01-26 22:53:25 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for custom_icons which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
```

```
2020-01-26 22:53:42 INFO (MainThread) [homeassistant.setup] Setting up custom_icons
2020-01-26 22:53:42 WARNING (MainThread) [custom_components.custom_icons.parser] no icons loaded for group 'custom'
```

I am sure I missed something :)

**Comment by petarvucetin** [comment]
Ok - missed this 
```
The custom_icons folder will be created at the root of your Home Assistant folder
```

**Comment by Armaell** [comment]
Hi petarvucetin.
Single svg icon files are not supported. I'll admit my knowledge is limited in this field and I don't know how to prepare complex svg files.

I invite you to use fontello.com [to create an icon](https://github.com/Armaell/home-assistant-custom-icons-loader#create-a-icon-file) pack with your files. Fontello has the advantage to simplify the icons, allowing me to easily use them.

**Comment by petarvucetin** [comment]
I have 0 expertise in SVG but I did find an icon I wanted somewhere else and dropped it on Fontello but everyone of them was distorted after conversion and something about path was in error... 

I am not sure why cards could just not display what ever standard image formats are, like png.

I might see if I can add support to it.

**Comment by Armaell** [comment]
Yeah, the simplification does ... indeed ... remove any complex stuff used by svg files. Like, even color is too much...

I did try to see if I could directly contribute to HA, but I failed to install a working environnement, and the time to learn how it worked, I found it easier for me to just scrap a component by cannibalizing other components.

In the end, it does have way too many limitations. Maybe one day if I have more time, I'll try again. If someone doesn't beat me to it
